packages/@storefront/core/CHANGELOG.md

```diff
@@ -4,9 +4,14 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [Unreleased] [<release type>]
+
```

Comment:
I don't think we have a newline here in our format, but it doesn't really matter.